great_expectations/zep/sources.py

```diff
@@ -3,33 +3,34 @@
 import logging
 from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Type, Union
 
-from great_expectations.util import camel_to_snake
+from typing_extensions import ClassVar
+
 from great_expectations.zep.type_lookup import TypeLookup
 
 if TYPE_CHECKING:
     from great_expectations.data_context import DataContext as GXDataContext
+    from great_expectations.execution_engine import ExecutionEngine
     from great_expectations.zep.context import DataContext
     from great_expectations.zep.interfaces import DataAsset, Datasource
 
 SourceFactoryFn = Callable[..., "Datasource"]
 
-LOGGER = logging.getLogger(__name__)
-
+LOGGER = logging.getLogger(__name__.lstrip("great_expectations."))
```

Comment:
TODO: revert this logger name shortening. Was looking at logs a lot and this was helpful, but shouldn't be part of the merged code.